cmd/client.go

```diff
@@ -173,6 +175,60 @@ func (cli *Client) BackupDatabase(c *clipkg.Context) error {
 	return cli.errorOut(saveBodyAsFile(resp, c.Args().First()))
 }
 
+// ImportKey imports a key to be used with the chainlink node
+func (cli *Client) ImportKey(c *clipkg.Context) error {
+	cfg := cli.Config
+	if !c.Args().Present() {
+		return cli.errorOut(errors.New("Must pass in filepath to key"))
+	}
+
+	src := c.Args().First()
+	kdir := cfg.KeysDir()
+
+	if err := keyStoreEmpty(kdir); err != nil {
+		return cli.errorOut(err)
+	}
+
+	if i := strings.LastIndex(src, "/"); i < 0 {
+		kdir += "/" + src
+	} else {
+		kdir += src[strings.LastIndex(src, "/"):]
+	}
+	return cli.errorOut(copyFile(src, kdir))
+}
+
+func keyStoreEmpty(kdir string) error {
```

Comment:
I think this is cleaner let me know what you think.

Reply:
I prefer returning the two values, whether it's empty and whether is has an error, because it makes the function more reusable. Really we just want to check if the directory is empty, that's the check we need. Along the way we may encounter an error, and due to Go's restrictions we have to either return the error as a second value or silently fail(not really an option IMO).